public/stylesheets/dashboard.css

```diff
@@ -0,0 +1,19 @@
+.cc-leftPanel {
```

Comment:
Let's put this in it's own left-panel.css file. I think it would be easier to find that way